dev/service-registry/service-locator/src/test/scala/com/lightbend/lagom/registry/impl/InternalRouterSpec.scala

```diff
@@ -32,7 +32,7 @@ class InternalRouterSpec extends FlatSpec with Matchers {
 
     router.rebuild(registry)
 
-    router.routeFor(Route("GET", "/pathABC", None)) should be(Found(httpUri))
+    // router.routeFor(Route("GET", "/pathABC", None)) should be(Found(httpUri))
```

Comment:
This test is failing now. 

We need to review the approach. Because `https` can also be found without a `portName`, that line may return `http` or `https`. 

I guess the solution must be: if you have a service that has both `http` and `https`, then we should declare it twice, like `my-service` and `my-service-https`. 

This is only applicable for external services. When we declare a real Lagom service, we only have one name, the one define in the `ServiceDescriptor` and it is `http`.